src/api/app/views/webui/request/beta_show.html.haml

```diff
@@ -16,91 +16,93 @@
+        .col-md-4.order-md-2.order-sm-1.mb-4
+          = render partial: 'webui/request/beta_show_tabs/conversation_aside',
+                  locals: { bs_request: @bs_request,
+                    my_open_reviews: @my_open_reviews,
+                    request_reviews: @request_reviews,
+                    package_maintainers: @package_maintainers,
+                    project_maintainers: @project_maintainers,
+                    show_project_maintainer_hint: @show_project_maintainer_hint }
 
-        .col-md-8.order-md-1.order-sm-2#comments-list
-          %h4.list-group.mb-4 Comments & Request History
-          .timeline.pb-2.ms-3{ data: { comment_counter: local_assigns[:comment_counter_id] } }
-            .timeline-item
-              .d-inline-flex
-                %i.fas.fa-lg.fa-code-commit.text-dark
-                - creator = User.find_by_login(@bs_request.creator) || User.nobody
-                = image_tag_for(creator, size: 35, custom_class: 'rounded-circle bg-light border border-gray-400 avatars-counter')
-                %p
-                  = link_to(realname_with_login(creator), user_path(creator))
-                  created this request
-                  = link_to('#request-creation', title: I18n.l(@bs_request.created_at.utc), name: 'request-creation') do
-                    = render TimeComponent.new(time: @bs_request.created_at)
-                  - if @bs_request.superseding.any?
-                    superseding
-                    - @bs_request.superseding.each do |superseded_request|
-                      = link_to("request ##{superseded_request.number}", request_show_path(superseded_request.number, anchor: 'overview'))
+        .col-md-8.order-md-1.order-sm-2
+          .row
+            #comments-list
+              .timeline.pb-2.ms-3{ data: { comment_counter: local_assigns[:comment_counter_id] } }
+                .timeline-item.mb-4.border.border-gray-200.rounded-3.bg-white.p-4#description-text
+                  - if @bs_request.description.present?
+                    %h4.list-group.mb-2 Description
+                    = render partial: 'webui/shared/collapsible_text', locals: { text: @bs_request.description, extra_css_classes: 'full-width' }
+                  - else
+                    %i No description set
 
-            - @bs_request.superseding.each do |superseding_request|
-              .timeline-item.pb-4
-                = link_to('#collapse-superseding', class: 'd-flex align-items-center',
-                    data: { 'bs-toggle': 'collapse' },
-                    aria: { expanded: false, controls: 'collapse-superseding' }) do
-                  %i.timeline-break.expander>
-                  %i.timeline-offset.collapser>
+                %h4.list-group.mb-4.ms-3 Comments & History
+                .timeline-item
+                  .d-inline-flex
+                    %i.fas.fa-lg.fa-code-commit.text-dark
+                    - creator = User.find_by_login(@bs_request.creator) || User.nobody
```

Comment:
This can be pushed as a `User` method

Reply:
It is true, definitely, it's a good point, but I wouldn't change the business logic here.

Unfortunately the changes comparison reports much more than the real changes, indeed that is something that was there already before. What this PR does is just to move few things around in the UI and decorate the request description with a border box. 

The scope of the PR is to improve the look&feel so I'd leave it for a later refactoring maybe. Do you agree?

Reply:
Yes, a follow-up with that small refactoring would be nice!